_blog.yml

```
@@ -3203,3 +3203,15 @@
     - guide
     - audio
     - transformers
+
+- local: amused
+  title: "Welcome aMUSEd: Efficient Text-to-Image Generation"
+  author: Isamu136
+  guest: true
+  thumbnail: /blog/assets/amused/thumbnail.png
+  date: Jan 02, 2024 (TODO: update)
```

```suggestion
  date: January 02, 2024
```
TODO: update

Attempting to fix the yaml linter.